scripts/bootstrap.sh.in

```diff
@@ -11,6 +11,7 @@ RPM=${RPM:-$(command -v rpm)}
 SYSTEMCTL=${SYSTEMCTL:-$(command -v systemctl)}
 YUM=${YUM:-$(command -v yum)}
 SALT_CALL=${SALT_CALL:-salt-call}
+SALT_MASTER_CALL=""
```

Comment:
Don't set it, such that `set -u` kicks in if we were to use the variable before it's configured properly.